floor/run-show.py

```diff
@@ -17,7 +17,7 @@
 SCRIPT_DIR = os.path.dirname(os.path.realpath(__file__))
 CONFIG_DIR = os.path.join(SCRIPT_DIR, 'config')
 PLAYLIST_DIR = os.path.join(CONFIG_DIR, 'playlists')
-MIDI_MAPPING_DIR = os.path.join(CONFIG_DIR, '/midi_maps')
+MIDI_MAPPING_DIR = os.path.join(CONFIG_DIR, 'midi_maps')
```

Comment:
If you were os.path.join and you were asked to join "/base/path" with "/path/part", you might do one of two things:

* Dumbly join on "/" and get an unsightly double slash in the path ("/base/path//path/part") that the OS will simply ignore
* Notice that your second arg begins with a slash, and simply join them to get what is intended ("/base/path/path/part") 

However you'd be wrong.  The correct answer, according to  os.path.join, is to assume the second arg is an absolute path, decide an absolute path cannot have a prefix, and silently drop the first arg, proudly returning only the second arg, "/path/part"

Reply:
😤